src/components/tutorial-meta/components/badges/helpers.ts

```diff
@@ -0,0 +1,71 @@
+import React, { ForwardRefExoticComponent, RefAttributes } from 'react'
+import { IconProps } from '@hashicorp/flight-icons/svg-react/types'
+import { IconClock16 } from '@hashicorp/flight-icons/svg-react/clock-16'
+import { IconPlay16 } from '@hashicorp/flight-icons/svg-react/play-16'
+import { IconTerminalScreen16 } from '@hashicorp/flight-icons/svg-react/terminal-screen-16'
+import { EditionOption } from 'lib/learn-client/types'
+import { ProductUsed as ClientProductUsed } from 'lib/learn-client/types'
+import {
+  getBadgeComponent,
+  ProductDisplayOption,
+  BadgeComponentProps,
+} from './components/badge'
+import { BadgeOptions } from '.'
+
+type DisplayOptions = {
+  [K in keyof BadgeOptions]: DisplayOption | ProductDisplayOption[]
+}
+
+type DisplayOption = {
+  label: string
+  icon?: ForwardRefExoticComponent<IconProps & RefAttributes<SVGSVGElement>>
```

Comment:
Not sure the best way to type an imported svg element component like this. This was what TS was suggesting 🤷‍♀️

Reply:
You should be able to use `ReactElement` like `StandaloneLink` does here: https://github.com/hashicorp/dev-portal/blob/main/src/components/standalone-link/types.ts

Reply:
Hmm yeah `ReactElement` gives me a Ts error, this was the only way I could make it happy

Reply:
Oh, I see. It's because you're setting the `icon` property in objects differently than how `StandaloneLink` accepts its `icon` prop as a React component. This will probably be easiest to refactor after the `Badge` component exists, so I think this'll be fine until then.